src/Containers/VendorsList/Profile.tsx

```diff
@@ -49,19 +51,23 @@ export const Profile: React.FC<IProfileProps> = ({
         case image === undefined:
             return (
                 <FeaturedProfile 
-                    {...profileProps}
-                    background={colors.primary}
+                    background={theme.colors.primary}
                     initials={profileInitials}
                     key={key} 
+                    width={PROFILE_WIDTH}
+                    height={PROFILE_HEIGHT}
+                    {...profileProps}
                 />
             )
         default:
             return (
                 <FeaturedProfile 
-                    {...profileProps}
                     image={image}
                     background='none'
                     key={key} 
+                    width={PROFILE_WIDTH}
+                    height={PROFILE_HEIGHT}
+                    {...profileProps}
                 />
```

Comment:
There are duplicate props here, abstract the props into an object then modify that object and spread it across both of these / just have one return statement. lots of cleaner code can be done here